tests/unit/specs/components/common/TmSelectNetwork.spec.js

```diff
@@ -52,51 +52,9 @@ describe(`TmSelectNetwork`, () => {
       }
     })
     self.networkId = `localnet`
-    wrapper.setData({
-      // network id is not set there
-      sortedNetworks: TmSelectNetwork.methods.updateSelectedNetwork.call(
-        self,
-        networks
-      )
-    })
   })
 
   it(`has the expected html structure`, async () => {
     expect(wrapper.element).toMatchSnapshot()
   })
-
-  it(`returns the networks sorted by the following criteria: current first, then default then mainnets, then testnets`, () => {
```

Comment:
did u change this?

Reply:
[yes](https://github.com/luniehq/lunie/pull/3830/files/396b1f2dd91ef2d71929503ca49be7fb2e241b87#diff-7054cab35095f712da18c2b3232f118bL66). i thought it was overcomplicated and unnecessary. network list always appears the same way now.